c8run/internal/windows/windows.go

```diff
@@ -0,0 +1,66 @@
+//go:build windows
+
+package windows
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+)
+
+func (w *WindowsC8Run) OpenBrowser(name string) {
+	operateUrl := "http://localhost:8080/operate/login"
+	openBrowserCmdString := "start " + operateUrl
+	openBrowserCmd := exec.Command("cmd", "/C", openBrowserCmdString)
+	openBrowserCmd.SysProcAttr = &syscall.SysProcAttr{
+		// CreationFlags: 0x08000000 | 0x00000200, // CREATE_NO_WINDOW, CREATE_NEW_PROCESS_GROUP : https://learn.microsoft.com/en-us/windows/win32/procthread/process-creation-flags
+	}
+	fmt.Println(name + " has successfully been started.")
+	openBrowserCmd.Run()
+}
+
+func (w *WindowsC8Run) GetProcessTree(commandPid int) []*os.Process {
+	return process_tree(int(commandPid))
```

Comment:
not part of the PR but... the method `process_tree` may return `[]*os.Process{foo, bar, nil}`, which is what a caller might expect.

```
processList := make([]*os.Process, len(tree))
```

should be

```
processList := make([]*os.Process, 0, len(tree))
```

Reply:
what is the purpose of the zero in your second argument? what information does that convey?